### PR TITLE
[monorepo] Drop "cf" prefix

### PR DESCRIPTION
**Pull request by cf19drofxots**
### Description

This change removes the `Cf` prefix on the names of classes.

@mvanderh this doesn't touch the operations classes though.

### CODE REVIEW

File: packages/iframe-wallet-poc/iframe-wallet/index.js [hunk 1]
```diff
@@ -133,14 +133,14 @@ async function pickUser(pkey) {
 
 function getApps() {
   const openChannelAddress = Object.keys(
-    iframeWallet.currentUser.vm.cfState.channelStates
```
Comment (mvanderh) [inline packages/iframe-wallet-poc/iframe-wallet/index.js]: This one was particularly silly, having a *cf*state on an obviously CF-branded object

File: packages/machine/src/vm.ts [hunk 2]
```diff
@@ -28,21 +28,21 @@ export class CfVmConfig {
...
+    state.stateChannelFromMultisigAddress(multisigAddress),
 
-  toAddress: (cfState: CfState, toAddress: cf.utils.Address) =>
-    cfState.stateChannelFromAddress(toAddress)
+  toAddress: (state: State, toAddress: cf.utils.Address) =>
+    state.stateChannelFromAddress(toAddress)
 };
 
-export class CounterfactualVM implements Observable {
+export class VM implements Observable {
```
Comment (mvanderh) [inline packages/machine/src/vm.ts]: Rename this to VirtualMachine?

Comment (snario) [reply packages/machine/src/vm.ts]: InstructionExecutor

File: packages/machine/src/state.ts [hunk 3]
```diff
@@ -3,9 +3,9 @@ import lodash from "lodash";
 
 import { deserialize } from "./serializer";
 import { OpCodeResult } from "./types";
-import { CounterfactualVM } from "./vm";
+import { VM } from "./vm";
 
-export class CfState {
+export class State {
```
[the remark this answers is not in the document]
Comment (snario) [reply packages/machine/src/state.ts]: I think this entire class is due for a refactor.

[the remark this answers is not in the document]
Comment (cf19drofxots) [reply packages/machine/src/state.ts]: Follow up PR.

**Pull request merged**